### DISCUSSION TITLE
Loopback 3 API docs are gone

### DISCUSSION THREAD

**Discussion by LiamDobbelaere** [loopbackio/loopback-next#9239] [Q&A]
The API docs for Loopback 3 are down, and I found this on Google Groups, however I just get an error message:
![image](https://user-images.githubusercontent.com/13522621/213423629-37afc422-5945-479b-8ac8-f41ca39eb85a.png)
![image](https://user-images.githubusercontent.com/13522621/213423663-4b3bbf30-13ea-4164-9e6b-ba763c3a8c5f.png)

I already tried checking out either master or tag v3.8.0 (our Loopback version) and using jsdoc 3 (the jsdoc version at the time that Google Groups answer was given) instead of 4.

We're still heavily dependent on Loopback 3 with no real possibility to upgrade anytime soon, so it's very regretful that these docs are no longer available for eg. newcomers in our team.

**Comment by shubhamp-sf** [comment]
Hi @LiamDobbelaere, I know it's too late but I think you missed a dot.

I've got it running for lb3 users here: https://loopback3-api-docs.vercel.app/

**Reply by shubhamp-sf** [reply]
Did the same for latest juggler code https://loopback-datasource-juggler-api-docs.vercel.app/
